_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,142 @@
+
+```json
+ "buckets": [
+    {
+      "min": 0.0,
+      "max": 5.0,
+      "count": 2
+    },
+    {
+      "min": 5.0,
+      "max": 10.0,
+      "count": 5
+    }
+  ]
+```
+
+You can create detailed representations of histogram buckets and their boundaries. You can control this feature by using the following parameters in a YAML file:
+
+```yaml
+  processor:
+    - otel_metrics_raw_processor:
+        calculate_histogram_buckets: true
+        calculate_exponential_histogram_buckets: true
+        exponential_histogram_max_allowed_scale: 10
+        flatten_attributes: false
+```
+
+Each array element describes one bucket. Each bucket contains the lower boundary, upper boundary, and its value count. This is a specific form of denser OpenTelemetry representation that is a part of the JSON output created by the following plugin:
+
+<!--- Is this correct? Is this a plugin? It looks like a JSON file.--->
+
+```json
+ "explicitBounds": [
+    5.0,
+    10.0
+  ],
+   "bucketCountsList": [
+    2,
+    5
+  ]
+```
+
+
+If `calculate_exponential_histogram_buckets` is not set to `false`, the following JSON values are added to each JSON historgram:
+
+```json
+
+    "negativeBuckets": [
+        {
+        "min": 0.0,
+        "max": 5.0,
+        "count": 2
+        },
+        {
+        "min": 5.0,
+        "max": 10.0,
+        "count": 5
+        }
+    ],
+...
+    "positiveBuckets": [
+        {
+        "min": 0.0,
+        "max": 5.0,
+        "count": 2
+        },
+        {
+        "min": 5.0,
+        "max": 10.0,
+        "count": 5
+        }
+    ],
+```
+
+The following JSON file is a more detailed form of the dense OpenTelemetry representation, which consists of negative and positive buckets, a scale parameter, offset, and list of bucket counts. 
+
+
+```json
+    "negative": [
+        1,
+        2,
+        3
+    ],
+    "positive": [
+        1,
+        2,
+        3
+    ],
+    "scale" : -3,
+    "negativeOffset" : 0,
+    "positiveOffset" : 1
+```
+
+The `exponential_histogram_max_allowed_scale` parameter defines the maximum allowed scale for an exponential histogram. Increasing this parameter increases potential memory consumption. See the [OpenTelemetry specifications](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto) for more information on exponential histograms and their computational complexity.
+
+All exponential histograms that have a scale that is above the configured parameter (by default, a value of `10`) are discarded and logged with an error level. 
```

Comment:
What is an error level and what happens when something is logged with it?

Reply:
This means Data Prepper will create a log message marked with ERROR. If a user checks the Data Prepper log, the user will see the message.